libs/community/tests/unit_tests/retrievers/test_bm25.py

```diff
@@ -43,3 +43,21 @@ def test_repr() -> None:
     ]
     bm25_retriever = BM25Retriever.from_documents(documents=input_docs)
     assert "I have a pen" not in repr(bm25_retriever)
+
+
+@pytest.mark.requires("rank_bm25")
+def test_k() -> None:
+    input_docs = [
+        Document(page_content="I have a pen."),
+        Document(page_content="Do you have a pen?"),
+        Document(page_content="I have a bag."),
+        Document(page_content="Do you have a dog?"),
+        Document(page_content="I have a cat."),
+    ]
+    bm25_retriever = BM25Retriever.from_documents(documents=input_docs)
+
+    documents = bm25_retriever.get_relevant_documents(query="I have a pen")
```

Comment:
I don't believe this is the right function signature, I think _get_relevant_docs is only meant to be called by the retriever class itself